README.md

```diff
@@ -1,69 +1,101 @@
+| Online volumetric CNMF | [demo_online_3D.ipynb](./demos/notebooks/demo_online_3D.ipynb) | Developed by Johannes Friedrich | 
+| Online CNMFE (OnACID-E) | [demo_realtime_cnmfE.ipynb](./demos/notebooks/demo_realtime_cnmfE.ipynb) |[Friedrich et al. 2020](https://pubmed.ncbi.nlm.nih.gov/33507937/) | 
+| Motion correction | [demo_motion_correction.ipynb](./demos/notebooks/demo_motion_correction.ipynb) | [Pnevmatikakis et al., 2017](https://pubmed.ncbi.nlm.nih.gov/28782629/) | 
+| Register cells across sessions | [demo_multisession_registration.ipynb](./demos/notebooks/demo_multisession_registration.ipynb) | [Pnevmatikakis et al., 2016](https://pubmed.ncbi.nlm.nih.gov/26774160/) | 
 
-More detailed docs on installation can be found [here](./docs/source/Installation.rst).
+A comprehensive list of references, where you can find detailed discussion of the methods and their development, can be found [here](https://caiman.readthedocs.io/en/master/CaImAn_features_and_references.html#references). 
 
-After installing the software, the caimanmanager.py script (which will be put in your path on Linux and OSX) is used to unpack datafiles and demos into a directory called caiman\_data. Invoke it with the "install" argument.
 
-If you are on Windows, the mechanism to put caimanmanager.py in your path may not work; in this case, you can locate it by typing python to bring up an interactive shell, and then
-```bash
->>> import sysconfig
->>> sysconfig.get_path('scripts')
-```
-You can invoke it as follows:
-```bash
-python C:/PATH/ABOVE/caimanmanager.py install
-```
+# Questions, help, and next steps 
+In addition to the demos, there are many ways to learn more about Caiman and receive help:
+- [The online documentation](https://caiman.readthedocs.io/en/latest/) contains a lot of general information about Caiman, the parameters, how to interpret its outputs, and more. 
+- The [Gitter forum](https://app.gitter.im/#/room/#agiovann_Constrained_NMF:gitter.im) is a great place to ask questions.
+- If you have found a bug, we recommend searching the [issues at github](https://github.com/flatironinstitute/CaImAn/issues) and opening a new issue if you can't find the solution there. 
+- If there is a feature you would like to see implemented, feel free to come chat at Gitter or open an issue at Github.
+
+In generaly, you don't have to need help to get involved. We love when users talk about use cases answer other people's questions, etc at Gitter and Github. 
+
+# How to contribute :hammer:
+ Caiman is an open-source project and improves because of contributions from users all over the world. If there is something about Caiman that you would like to improve, then you are qualified to contribute! We are always looking for more contributors, so please come read the [contributors page](./CONTRIBUTING.md) for more details about how. 
```

Comment:
This is a little bit too breathless. Not everyone who wants to contribute will actually have the skills to do so.

Reply:
Contributing can mean participating at Gitter, opening issues, all sorts of things. The goal here is to stimulate participation not tentativeness, which tends to be the default state.